.changeset/wise-feet-retire.md

```diff
@@ -0,0 +1,5 @@
+---
+"@onflow/fcl-wc": patch
+---
+
+Create FCL WC adapter package
```

Comment:
Is this changeset redundant?  Isn't it already covered by [.changeset/tame-bags-kneel.md](https://github.com/onflow/fcl-js/pull/1352/files#diff-bfcf6603abe72b17b1cf66818c64cc73d56a0336d981d1ceefec5d2e5ea1be8f).

Reply:
sure is, leftover from merge is guess. Can just delete right?